### PR TITLE
Cleaning code

### PR DESCRIPTION
**Pull request by cleorenaud**
With this PR, some comments have been added to diverse functions and composables of the app.
Unused functions have been removed

### CODE REVIEW

File: app/src/androidTest/java/com/example/triptracker/map/MapOverviewTest.kt
Comment (Jeremmmyyyyy) [inline app/src/androidTest/java/com/example/triptracker/map/MapOverviewTest.kt:1]: Thanks for the Uppercase formatting

File: app/src/main/java/com/example/triptracker/authentication/GoogleAuthenticator.kt
Comment (Jeremmmyyyyy) [inline app/src/main/java/com/example/triptracker/authentication/GoogleAuthenticator.kt:1]: NIce additions on the missing comments

File: app/src/main/java/com/example/triptracker/model/location/Pin.kt [hunk 1]
```diff
@@ -7,20 +7,21 @@ package com.example.triptracker.model.location
...
- * @param image_url Image url of pin
+ * @param imageUrl Image url of pin
  */
 data class Pin(
     val latitude: Double,
     val longitude: Double,
     val name: String,
     val description: String,
-    val image_url: List<String>
+    val imageUrl: List<String>
```
Comment (Jeremmmyyyyy) [inline app/src/main/java/com/example/triptracker/model/location/Pin.kt:17]: Nice catch

File: app/src/main/java/com/example/triptracker/model/location/PopUpState.kt
Comment (Jeremmmyyyyy) [inline app/src/main/java/com/example/triptracker/model/location/PopUpState.kt:1]: This is much better now

File: app/src/main/java/com/example/triptracker/model/repository/ItineraryRepository.kt [hunk 2]
```diff
@@ -118,57 +101,14 @@ open class ItineraryRepository {
...
-  //  }
 
   /**
    * Add a new itinerary to the database
    *
    * @param itinerary Itinerary object to add Serialize the Itinerary object to a map and add it to
    *   the database
    */
-  suspend fun addNewItinerary(itinerary: Itinerary) {
+  fun addNewItinerary(itinerary: Itinerary) {
```
Comment (Jeremmmyyyyy) [inline app/src/main/java/com/example/triptracker/model/repository/ItineraryRepository.kt:111]: Why is the suspend removed ?

**Review by Jeremmmyyyyy — APPROVED** [review]
LGTM ! Nice changes. This improves the code readability and cleans the whole codebase which is nice for the final release. Thanks for it !

File: app/src/main/java/com/example/triptracker/model/repository/ItineraryRepository.kt (hunk 2 above)
Comment (cleorenaud) [reply app/src/main/java/com/example/triptracker/model/repository/ItineraryRepository.kt:111]: It was displayed as being redundant and unnecessary

**Pull request merged**